211018_4win_main.py

```diff
@@ -49,28 +49,21 @@ def digonal_win_left_top(player):
 
 def diagonal_win_left_bottom(player):
     count = 0
-    for column in range(6):
-        n = -2
-        for row in range(6):
-            if column + n == 0:
-                count = 0
-            try:
-                if matrix[row][column + n] == player:
-                    print("+")
-                    count += 1
+    for column in range(7):
+        for row in range(5, 0 - 1, -1):             # begin to look after a player token on position "50"

```

Comment:
I dont see why use "range(5, 0 - 1, -1)"
"range(6)" yield the same results and is more readable

Reply:
Yeah, it does.
I didn't thought about it, as I wrote this part of the code